Pear/Profile/ProfilePromptTableViewCell.swift

```diff
@@ -54,19 +54,19 @@ class ProfilePromptTableViewCell: UITableViewCell {
         }
     }
 
-    func configure(for user: User, type: ProfileSectionType) {
+    func configure(for user: UserV2, type: ProfileSectionType) {
         titleLabel.text = type.getTitle(for: user)
         if type == .basics {
             descriptionTextView.attributedText =
                 NSMutableAttributedString()
                 .thinFont("I study ")
-                .normalFont(user.major)
+//                .normalFont(user.major)
                 .thinFont(" in the class of ")
-                .normalFont(user.graduationYear)
+                .normalFont(user.graduationYear ?? "2020")
```

Comment:
any reason we're defaulting to 2020? I think that can potentially mask a bug in the future if we're not getting any years from the backend and then make it tougher to debug